src/plugin.py

```diff
@@ -144,9 +154,58 @@ async def get_owned_games(self):
             return [g.in_galaxy_format() for g in self._owned_games.values()]
 
     async def get_subscriptions(self):
-        return [
-            Subscription(SUBSCRIPTIONS.TROVE, subscription_discovery=SubscriptionDiscovery.USER_ENABLED)
-        ]
+        subscriptions: List[Subscription] = []
+        current_month_unlocked = False
+        current_or_former_subscriber = await self._api.had_subscription()
+
+        if current_or_former_subscriber:
+            async for product in await self._api.get_subscription_products_with_gamekeys():
+                if not product.get('isChoiceTier'):
+                    # no more choice content, now humble montly data coming from before 2020
+                    break
+                subscriptions.append(
+                    Subscription(product['title'], owned=True)
+                )
+                if product.is_active_content:
+                    # assuming only current month has "is_active_content": true
+                    current_month_unlocked = True
+
+        if not current_month_unlocked:
+            '''
+            - for not subscribers as potential discovery of current choice games
+            - for subscribers who has not used "Early Unlock" yet:
+              https://support.humblebundle.com/hc/en-us/articles/217300487-Humble-Choice-Early-Unlock-Games
+            '''
+            choice_months_details = await self._api.get_choice_month_details()
+            active_month = choice_months_details['active_month']
+
+            if current_or_former_subscriber:
+                active_month_content = await self._api.get_choice_content_data(
+                    active_month['montly_product_page_url']
+                )
+                if active_month_content.user_subscription_plan is not None:
+                    current_month_unlocked = True
```

Comment:
second, different meaning come here. So overused var with 2 meanings: current_month_unlocked and (here) going_to_be_unlocked.

Separate or keep rename to sth more general like `is_subscription_active`/ `active_plan`/`has_active_plan`

Reply:
And check what is under `active_month_content.user_subscription_plan` after cancelling subscr!

Reply:
check how long trove lasts after cancelling sub